pkg/gardenlet/controller/managedseed/managedseed_controller_test.go

```diff
@@ -67,7 +69,7 @@ var _ = Describe("Controller", func() {
 			ObjectMeta: metav1.ObjectMeta{
 				Name:       name,
 				Namespace:  namespace,
-				Generation: 1,
+				Generation: 2,
```

Comment:
why didn't you leave that at `1` and test it with accordingly ?

Reply:
Addressed in [commit](https://github.com/gardener/gardener/pull/5483/commits/de284d6ecaa02c1411dda2dee6accfbb949d97f5).

Reply:
If you implemented the request and don't have further questions you can go ahead and resolve the converstations.